modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -8,25 +8,34 @@ If you plan to install Mule and run it on premises, review these minimum hardwar
 
 == Minimum Hardware Requirements
 
-* 2 GHz CPU or 1 virtual CPU in virtualized environments
-* 1 GB of RAM. Add RAM to match your latency requirements, or to support large simultaneous messages or a large number of simultaneous messages if your applications process them.
-* 4 GB of storage
+* 2 GHz CPU or 1 virtual CPU in virtualized environments.
+* 1 GB of RAM.
+* 4 GB of storage.
 
-== Supported Software and Systems
+Adjust RAM to match your latency requirements and the size and number of simultaneous messages that applications process.
 
-Verify that you use a supported version of Java and a supported operating system before you install Mule.
+== Required Software
 
-Mule Runtime runs as a standalone server; therefore an application server is not required. However, if you use an application server or need to access a data store, verify that you use a supported application server or database.
+Verify that you use a supported version of Java before you install Mule.
 
 [%header,cols="15a,85a"]
 |===
 |Software |Version
 | JDK | JDK 1.8.0 (1.8.0_202 recommended)
+|===
+
+== Supported Software
+
+Mule runs on any operating systems that runs a supported Java Runtime Environment (JRE). The Mule runtime engine passed functional testing against the following software and operating systems:
+
+|===
+|Software |Version
 | OS | MacOS 10.11.x, HP-UX 11i V3, AIX 7.2, Windows 2016 Server (recommended), Windows 2012 R2 Server, Windows 10 (recommended), Windows 8.1, Solaris 11.3, RHEL 7, Ubuntu Server 18.04, Linux Kernel 3.13+
-| Databases | Oracle 11g, Oracle 12c, MySQL 5.5+, IBM DB2 10, PostgreSQL 9, Derby 10, Microsoft SQL Server 2014
-| Browsers for working on Anypoint Platform | The latest versions of Firefox, Chrome, and Edge. Note that IE is being deprecated. Minimum screen resolution of 1024x768
+| JDK | JDK 11 
```

Comment:
We also tested against JDK 1.8, should we include it here too?

Reply:
Updated.